### DISCUSSION TITLE
Oobabooga Multi User Issue

### DISCUSSION THREAD

**Discussion by Vishal-Raghav** [oobabooga/text-generation-webui#3218] [General]
Hi Team,
I used (--multi-user) in the command line parameters and used NGROK to publish the service. I tested from different laptops. The speed of response is great, however, the UI doesn't accept a new request while the model is responding to other existing request. It's like first come first go service. Is there some that I can do to make the model work in parallel so that all requests from various laptops are answered simultaneously.

**Comment by sandwichdoge** [comment]
+1
Batched inferencing would be nice for collab.
